packages/composites/bitcoin-json-rpc/src/endpoint/scantxoutset.ts

```diff
@@ -0,0 +1,35 @@
+import JSONRPC from '@chainlink/json-rpc-adapter'
+import { Config, ExecuteWithConfig } from '@chainlink/types'
+import { Validator, Requester } from '@chainlink/ea-bootstrap'
+
+export const NAME = 'scantxoutset'
+
+const inputParams = {
+  scanobjects: ['addresses', 'scanobjects'],
+  confirmations: false
+}
+
+export const execute: ExecuteWithConfig<Config> = async (request, config) => {
+  const validator = new Validator(request, inputParams)
+  if (validator.error) throw validator.error
+
+  const jobRunID = validator.validated.id
+  const scanobjects = validator.validated.data.scanobjects.map((address: string) => {
+    // Addresses must be formatted as addr(39e7mxbeNmRRnjfy1qkphv1TiMcztZ8VuE)
+    if (address.substr(0, 4) == 'addr') return address
```

Comment:
Should we enforce this more comprehensively?

Such as with something like `/^addr\(\w*\)$/g.test(address)`.
Or even test valid character lengths of BTC addresses.